docs/release-source/release/spies.md

```diff
@@ -470,3 +385,5 @@ Returns the passed format string with the following replacements performed:
 </dl>
 
 [call]: ../spy-call
+
+{% include embed-scripts.html %}
```

Comment:
By moving this into the `footer.html`, after with the script to load runkit, you avoid blocking rendering of the content above and avoid crashing at the same time.
```suggestion
```